service/src/main/java/bio/terra/landingzone/stairway/flight/LandingZoneDefaultParameters.java

```diff
@@ -0,0 +1,66 @@
+package bio.terra.landingzone.stairway.flight;
+
+import static bio.terra.landingzone.library.landingzones.definition.factories.CromwellBaseResourcesFactory.STORAGE_ACCOUNT_BLOB_CORS_ALLOWED_HEADERS_DEFAULT;
+import static bio.terra.landingzone.library.landingzones.definition.factories.CromwellBaseResourcesFactory.STORAGE_ACCOUNT_BLOB_CORS_ALLOWED_METHODS_DEFAULT;
+import static bio.terra.landingzone.library.landingzones.definition.factories.CromwellBaseResourcesFactory.STORAGE_ACCOUNT_BLOB_CORS_ALLOWED_ORIGINS_DEFAULT;
+import static bio.terra.landingzone.library.landingzones.definition.factories.CromwellBaseResourcesFactory.STORAGE_ACCOUNT_BLOB_CORS_EXPOSED_HEADERS_DEFAULT;
+import static bio.terra.landingzone.library.landingzones.definition.factories.CromwellBaseResourcesFactory.STORAGE_ACCOUNT_BLOB_CORS_MAX_AGE_DEFAULT;
+
+import bio.terra.landingzone.library.landingzones.definition.factories.CromwellBaseResourcesFactory;
+import bio.terra.landingzone.library.landingzones.definition.factories.parameters.StorageAccountBlobCorsParametersNames;
+import com.azure.resourcemanager.containerservice.models.ContainerServiceVMSizeTypes;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+// TODO: spread parameters among flight steps
```

Comment:
Does it make sense to go ahead and do this now?

Reply:
Or if this is the same code as https://github.com/DataBiosphere/terra-landing-zone-service/blob/47d13216db8d289c5414499fb18adb5f4bac8b62/service/src/main/java/bio/terra/landingzone/library/landingzones/definition/factories/CromwellBaseResourcesFactory.java#L389, can you make CromwellBaseResourcesFactory use this class instead so that the code is not duplicated?

Reply:
I would leave it as is for now. Agree that there is code duplication, but most likely the part from CromwellBaseResourcesFactory will gone once we switch to stairway-base flight.